src/bootstrap/plugin.loader.ts

```diff
@@ -22,8 +22,9 @@ import { SlowModePlugin } from '../app/plugins/slowmode.plugin';
 import { ScoresPlugin } from '../app/plugins/scores.plugin';
 import { MarketPlacePlugin } from '../app/plugins/marketplace.plugin';
 import { ManageRolesPlugin } from '../app/plugins/manageroles.plugin';
+import { StatusPlugin } from '../app/plugins/status.plugin';
 
-const PluginStore: { [pluginName: string]: any } = {
+export const PluginStore: { [pluginName: string]: any } = {
```

Comment:
why adding `export` now

Reply:
I cant reach it from the plugin unless its exported, unless there is another way

Reply:
we shouldn't be exposing the entire store globally when all we want to know is the size of it. this can potentially allow people to interact with the PluginStore after run-time which is what we don't want. it should remain immutable.

Reply:
we can either make a wrapper method that returns the size of the store via PluginLoader or keep the export and utilize Object.freeze()

Reply:
Just add a method in the plugin loader to return the size of the store?

Reply:
yup

Reply:
Can you get access to the plugin loader from this.container?